tests/test_slim_engine.py

```diff
@@ -1003,7 +1004,7 @@ def verify_mutation_rates(self, contig, ts):
             self.verify_rates_match(mut_rate, intervals, slim_rates, slim_ends)
 
         # also verify any bits not covered by DFEs have no mutations
-        empty_intervals = np.empty((0, 2))
+        empty_intervals = np.array([[0, int(contig.length)]])
```

Comment:
Idk why I did this, but at one point the `_check_intervals_validity` was failing on an empty numpy array. either way, this seems more natural to me?

Reply:
Hm - well, this is not the same thing and the error was because `np.empty( )` needed `dtype='int'`... but, I think you're right! This was in error, and your array is the correct one.